README.md

```diff
@@ -2,6 +2,45 @@
 
-# Fake it till you make `Git`
+# _Fake it till you make_ `Git`
 
-An (opinionated) approach to `Git` and `GitHub`
+Welcome to "Fake it till you make `Git`" - a resource dedicated to educating people on how to use `Git` and `Github`. This repository is published via `Github Pages` and can be found on [this link](https://nicolaerario.github.io/fake-it-till-you-make-git/), and we hope to help `you` become proficient in using `Git` and `Github`.
+
+## _How to contribute_
+
+We welcome `contributions` to this resource from anyone who is interested in helping us `improve` and `expand` our content. If you're interested in contributing, please take a look at our Github repository [issues page](https://github.com/nicolaerario/fake-it-till-you-make-git/issues) and create a `new issue` or make a `pull request` for an existing one. We accept contributions in the form of `bug reports`, `website improvements`, and `new content`. We also welcome `translations` of our existing content into other languages.
+
+## _Create and Edit a_ `Resource Article`
+
+To create or edit an `article` on our Github repository, follow these steps:
+
+- Navigate to the "[docs](https://github.com/nicolaerario/fake-it-till-you-make-git/tree/develop/docs)" directory in the repository.
+- Create a new file named `title-of-the-article.md` all in lowercase and separated by the hyphens.
+- Add this at the start of the article:
+  layout | title | description
+  ------------- | ------------- | -------------
+  page | Your Article Title | A short description of the article
+- Write the article with anything you'd like to hear regarding the theme that you choose.
+- Submit a pull request to have your contribution reviewed.
+
+When creating an article, make sure to follow the `style` used on the rest of the website to ensure that the article is consistent with the rest of this resource.
+
+## _Create a_ `Blog Post`
+
+To create a `blog post` on our Github repository, follow these steps:
+
+- Navigate to the "[\_posts](https://github.com/nicolaerario/fake-it-till-you-make-git/tree/develop/_posts)" directory in the repository.
+- Create a new file named `YYYY-MM-DD-title-of-your-post.md` all in lowercase and separated by hyphens.
+- Add this at the start of the blog post:
+  layout | title | categories | author | meta
+  ------------- | ------------- | ------------- | ------------- | -------------
+  post | Your Post Title | temp | Your Name | temp
+- Submit a pull request to have your contribution reviewed.
+
+When creating your post, make sure to follow the style used on the rest of the website to ensure that your post is consistent with the rest of this resource.
+
+---
+
+## _Contacts_
+
+If you have any question or problem about this project you can always send me an [email](mailto:nicola.erario@gmail.com) and I will try to answer you ASAP :)
```

Comment:
For questions and problems I think to use `issues` at the moment. Remove other contacts